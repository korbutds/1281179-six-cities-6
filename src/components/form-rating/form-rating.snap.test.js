import React from 'react';
import {render} from '@testing-library/react';
import {Test} from '../../test/test-mock';
import FormRating from './form-rating';
test(`CommentForm component render correctly`, () => {
  const {container} = render(
      <Test>
        <FormRating />
      </Test>
  );
  expect(container).toMatchSnapshot();
});
