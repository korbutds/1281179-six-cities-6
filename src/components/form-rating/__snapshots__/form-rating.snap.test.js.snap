// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentForm component render correctly 1`] = `
<div>
  <div
    class="reviews__rating-form form__rating"
  >
    <input
      class="form__rating-input visually-hidden"
      id="5-stars"
      name="rating"
      required=""
      type="radio"
      value="5"
    />
    <label
      class="reviews__rating-label form__rating-label"
      for="5-stars"
      title="perfect"
    >
      <svg
        class="form__star-image"
        height="33"
        width="37"
      >
        <use
          xlink:href="#icon-star"
        />
      </svg>
    </label>
    <input
      class="form__rating-input visually-hidden"
      id="4-stars"
      name="rating"
      required=""
      type="radio"
      value="4"
    />
    <label
      class="reviews__rating-label form__rating-label"
      for="4-stars"
      title="good"
    >
      <svg
        class="form__star-image"
        height="33"
        width="37"
      >
        <use
          xlink:href="#icon-star"
        />
      </svg>
    </label>
    <input
      class="form__rating-input visually-hidden"
      id="3-stars"
      name="rating"
      required=""
      type="radio"
      value="3"
    />
    <label
      class="reviews__rating-label form__rating-label"
      for="3-stars"
      title="not bad"
    >
      <svg
        class="form__star-image"
        height="33"
        width="37"
      >
        <use
          xlink:href="#icon-star"
        />
      </svg>
    </label>
    <input
      class="form__rating-input visually-hidden"
      id="2-stars"
      name="rating"
      required=""
      type="radio"
      value="2"
    />
    <label
      class="reviews__rating-label form__rating-label"
      for="2-stars"
      title="badly"
    >
      <svg
        class="form__star-image"
        height="33"
        width="37"
      >
        <use
          xlink:href="#icon-star"
        />
      </svg>
    </label>
    <input
      class="form__rating-input visually-hidden"
      id="1-stars"
      name="rating"
      required=""
      type="radio"
      value="1"
    />
    <label
      class="reviews__rating-label form__rating-label"
      for="1-stars"
      title="terribly"
    >
      <svg
        class="form__star-image"
        height="33"
        width="37"
      >
        <use
          xlink:href="#icon-star"
        />
      </svg>
    </label>
  </div>
</div>
`;
